### ISSUE TITLE
Expose functionality as View > Active Editor option to run in real time instead of just function invocation

### ISSUE THREAD

**Issue by bronius** [abrookins/WrapToColumn#37]
Love this little thing, works for me in a php block comment just as I had hoped!

Would it be possible to have it run continuously such that, in effect, the word wrapping is bound to a block comment's dimensions as I type? It could then become another option among, "Soft wrap," "Show line numbers," etc.

**Comment by abrookins** [comment]
Interesting idea. Would it work like the "hard wrap" setting in Intellij editors, but effectively bind only to comment blocks?

**Comment by bronius** [comment]
Yes, that sounds exactly what I mean. I haven't a clue how this would affect your plugin design nor whether it might inadvertently "leak" over time and affect editor performance. :)
